src/nodes/Heading/HeadingView.vue

```diff
@@ -19,7 +19,7 @@
   -->
 
 <template>
-	<NodeViewWrapper :id="node.attrs.id"
+	<NodeViewWrapper :id="'heading-' + node.attrs.id"
```

Comment:
I think we should not change the ID only on the node view, but directly on the node props.
So probably here:
https://github.com/nextcloud/text/blob/33ae28d3444195ff2a18610eeaccaecde26d66d5/src/nodes/Heading/extractor.js#L27

So that the ID property of the node is the real node and one would not have to check multiple files to get the Dom id/ have multiple IDs.

Reply:
I was thinking the same initially. But then i figgured that this would also change the `href` attribute of the resulting tag.
I'd prefer to keep the `href` as is because `Our-Garden#tomatoes` looks better to me then `Our-Garden#heading-tomatoes`.

Reply:
That's the way how anchors work: The href **must** match the ID of the heading so that the browser scrolls to the element.

Reply:
Ohhh... you are right. I was confused there. :facepalm:

Reply:
> ... because `Our-Garden#tomatoes` looks better to me then `Our-Garden#heading-tomatoes`

If this is a problem, maybe then use a shorter prefix, like `h-` or similar.
But after releasing this we should try to **not** change the prefix as this would annoy many users, for example if you set up links to sections all links are now invalid any you have to change every link.

Reply:
I like the idea to change the prefix to `h-` to make it shorter. What do you think @snehar97? Would you be up to this change?